storage-resize-images/functions/src/resize-image.ts

```diff
@@ -141,6 +141,7 @@ export const modifyImage = async ({
       metadata: objectMetadata.metadata || {},
```

Comment:
```suggestion
      metadata: objectMetadata.metadata ? { ...objectMetadata.metadata } : {},
```

This will fix duplicate metadata across resized images